docs/developer/decision-records/2023-12-12-dataplane-signaling/README.md

```diff
@@ -0,0 +1,19 @@
+Data Plane Signaling and Access Control Architecture
```

Comment:
could you link this decision record in `docs/developer/decision-records/README.md`?